code/modules/power/solar.dm

```diff
@@ -158,7 +158,9 @@
 
 	sunfrac = cos(p_angle) ** 2
 
-#define SOLARGENRATE 1500
+// Previous SOLARGENRATE was 1500 WATTS processed every 3.3 SECONDS.  This provides 454.54 WATTS every second
+// Adjust accordingly based on machine proc rate
+#define SOLARGENRATE (454.54 * MACHINE_PROCS_PER_SEC)
```

Comment:
What did I say about nice numbers 👀? It doesn't have to match legacy behavior *exactly*.

Reply:
I unconsciously disregarded your statement about "nice numbers" and used the rest of your statement to reinforce my original concern that the way the PR was written comes off as enforcing the previous behavior was correct instead of distilling the defines and logic to allow for scaling and adjustment.  

Is there a downside to the specificity?  Honestly I don't care either way as they are floats (under the hood and in practice).  Original intended scope of PR was to reverting behavior.

Reply:
Fair. No real downside, per se.

I/you/someone else can always change up the numbers at a later date, so go for it.